<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercado</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>C:/Users/docaf/Downloads/Visualpharm-Finance-Cash-register.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>671</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="products_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>391</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
border-radius:10px;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="total">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>290</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 40px;
color: white;</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="save_product">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 10px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>CÃ³digo do produto</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="register_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>430</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="product_amount">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>430</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 5px;</string>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="product_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="nfe_button">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 252, 164);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Gerar nota fiscal</string>
     </property>
    </widget>
    <widget class="QListWidget" name="products_list2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>151</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
border-radius:10px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="discount_input">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>150</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
border-radius: 10px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Cupom de desconto</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="discount_button">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>150</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remove_button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Excluir </string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="product_qtd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="total_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
color: white;</string>
     </property>
     <property name="text">
      <string>Produto</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
color: white;</string>
     </property>
     <property name="text">
      <string>Quantidade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="github">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>420</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: none;
image: url(:/newPrefix/github.png);
font-size: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="a.qrc"/>
 </resources>
 <connections/>
</ui>
